website/blog/2022-11-21-react-native-core-contributor-summit-2022.md

```diff
@@ -0,0 +1,123 @@
+---
+title: React Native Core Contributor Summit 2022
+authors: [thymikee, cortinico]
+tags: [announcement]
+date: 2022-11-21
+---
+
+# React Native Core Contributor Summit 2022
+
+After years of pandemic and online-only events, we really felt it was time to bring the Core Contributors of React Native together!
+
+That’s why at the beginning of September, we gathered some of the active core contributors of React Native, library maintainers, and the Meta’s React Native and Metro teams to the **Core Contributor Summit 2022**. [Callstack](https://www.callstack.com/) hosted the Summit in their HQ in Wrocław, Poland, as a part of the [React Native EU](https://www.react-native.eu/) conference happening at the same time.
+
+Together with the React Native core team, we devised a series of **workshops** in which the attendees could participate. The topics were:
+
+- ​​React Native Codegen & TypeScript Support
+- ​​React Native New Architecture Library Migration
+- ​​React Native Monorepo
+- Metro Web and Ecosystem Alignment
+- Metro Simplified Release Workflow
+
+We were impressed by the amount of knowledge-sharing and collaboration over those two days. In this blog post, we’d like to give you a sneak peek of the results of this gathering.
+
+<!--truncate-->
+
+### React Native Codegen & TypeScript Support
+
+[React Native’s Codegen](https://reactnative.dev/docs/the-new-architecture/pillars-codegen) is a fundamental part of the New Architecture of React Native. Supporting and improving it is among our top priorities for the future of React Native. For instance, earlier this year, we added support for generic code starting from TypeScript specs rather than Flow.
```

Comment:
```suggestion
[React Native’s Codegen](/docs/the-new-architecture/pillars-codegen) is a fundamental part of the New Architecture of React Native. Supporting and improving it is among our top priorities for the future of React Native. For instance, earlier this year, we added support for generic code starting from TypeScript specs rather than Flow.
```